tests/test.py

```diff
@@ -33,6 +35,20 @@ def test_file_api(self):
         res = self.app.get("/v1/files/123?replica=aws")
         self.assertEqual(res.status_code, requests.codes.found)
 
+        res = self.app.put(
+            '/v1/files/123',
+            data=json.dumps(
+                dict(source_url="s3://foobar",
+                     bundle_uuid=str(uuid.uuid4()),
+                     creator_uid=4321,
+                     content_type="text/html",
+                     )),
```

Comment:
What do you think about writing a test helper mixin to be able to just say `res = self.assert_response('/v1/files/123', json=dict(...), code=requests.codes.created); self.assertEqual(res.json["foo"], bar)`?